### PR TITLE
Update syntax tree visible endpoint node with `packageName` and `version`

### PR DESCRIPTION
**Pull request by kanushka**
## Purpose
Update syntax tree visible endpoint node with `packageName` and `version`.

## Approach
Use module object's package name and version to update custom syntax tree.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by NipunaMarcus** [comment]
There are a few places not covered by the test you added ...  can you take a look?

File: misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java [hunk 1]
```diff
@@ -270,7 +274,9 @@ private JsonObject visibleEP(Node node, TypeSymbol typeSymbol, boolean isRemoteA
             symbolMetaInfo.addProperty("isCaller", "Caller".equals(typeSymbol.getName().orElse(null)));
```
Comment (nadeeshaan) [inline misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java:274]: Do not use `orElse(null)` this bypasses the advantage of optional, hence use `orElse("")`

File: misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java [hunk 2]
```diff
@@ -251,7 +251,9 @@ private JsonObject visibleEP(Node node, TypeSymbol typeSymbol, boolean isRemoteA
         JsonObject symbolMetaInfo = new JsonObject();
         ModuleID moduleID = typeSymbol.getModule().isPresent() ? typeSymbol.getModule().get().id() : null;
         String orgName = moduleID != null ? moduleID.orgName() : "";
+        String packageName = moduleID != null ? moduleID.packageName() : "";
         String moduleName = moduleID != null ? moduleID.moduleName() : "";
+        String version = moduleID != null ? moduleID.version() : "";
 
         if (node.kind() == SyntaxKind.REQUIRED_PARAM) {
```
Comment (nadeeshaan) [inline misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java:258]: If we consider the if-else ladder we have used here, there is a considerable amount of duplication for setting the properties such as `typeName`, `orgName` and etc. Let's refactor this one in this PR itself. Check whether if we can come up with a solution based on a design pattern as well (Builder pattern maybe).

**Review by nadeeshaan — CHANGES_REQUESTED** [review]

File: misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java (hunk 2 above)
Comment (kanushka) [reply misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/SyntaxTreeMapGenerator.java:258]: sure. will refactor this.

**Review by nadeeshaan — APPROVED** [review]
LGTM

**Pull request merged**